intern.json

```diff
@@ -10,16 +10,16 @@
 	],
 
 	"suites": [
-		"./_build/common/tests/unit/all.js"
+		"./dist/dev/src/common/tests/unit/all.js"
 	],
 
 	"functionalSuites": [
-		"./_build/common/tests/functional/all.js"
+		"./dist/dev/src/common/tests/functional/all.js"
 	],
 
 	"browser": {
 		"loader": {
-			"script": "./node_modules/grunt-dojo2/lib/intern/internLoader.js",
+			"script": "./node_modules/@dojo/scripts/intern/loader.js",
 			"options": {
 				"packages": [
 					{ "name": "src", "location": "_build" }
```

Comment:
Should this `_build` be `dist/dev/src` as well?

Reply:
probably... I don't know what this does tho lol

Reply:
Just `dist/dev`, although I don't see any place we're actually using this `browser` config.